Gemfile

```diff
@@ -7,7 +7,6 @@ ruby '2.3.0'
 gemspec
 
 group :development, :test do
-  gem 'rspec-collection_matchers'
```

Comment:
Perhaps a naive question, but why do we need rubocop in the Gemfile when all the rest is in the gemspec file?